source/_integrations/generic.markdown

```diff
@@ -15,7 +15,12 @@ Home Assistant will serve the images via its server, making it possible to view
 
 ## Configuration
 
-To enable this camera in your installation, add the following to your `configuration.yaml` file:
+Home Assistant offers the Generic IP Camera through **Configuration -> Integrations -> Generic IP Camera**.
```

Comment:
I think we can delete this and replace it with `{% include integrations/config_flow.md %}`

Reply:
Agreed. Done!